pilot/lib/pilot/tester_manager.rb

```diff
@@ -27,6 +27,32 @@ def add_tester(options)
       UI.success("Successfully added tester #{config[:email]} to app #{app.name} in group(s) #{group_names}")
     end
 
+    def add_tester_to_build(options)
+      start(options)
+      app = find_app(apple_id: config[:apple_id], app_identifier: config[:app_identifier])
+      UI.user_error!("You must provide either a Apple ID for the app (with the `:apple_id` option) or app identifier (with the `:app_identifier` option)") unless app
+
+      email_param = config[:email]
+      UI.user_error!("You must provide 1 or more emails (with the `:email` option)") unless email_param
+
+      if config[:build_number].nil?
+        build = app.get_builds[0]
+      else
+        builds = app.get_builds.select do |b|
+          config[:build_number] == b.version
+        end
+        UI.user_error!("Build #{config[:build_number]} not found.") unless builds.count == 1
+        build = builds[0]
+      end
+      UI.success("Build: #{build.version} id: #{build.id}")
+
+      app.get_beta_testers.select do |tester|
+        next unless email_param.include?(tester.email)
```

Comment:
Should we check for an exact (case insensitive) match instead of using `include`? This would return false positives in case `email_param` is `jpmorgan@email.com` and `tester.email` is `morgan@email.com`, for instance 😅

Reply:
Oh, I get it now... You used `include` to cover cases where the `:email` argument contains multiple emails, is that right?
Although this might work, it's not very intuitive imo, first because email is in singular but also because of the false positive I mentioned above 🤔 
What if this API allowed only single emails to be used, for now? Either that, or use a `:emails` (plural) field, that is a list of strings that can be properly parsed as so. Then, in this condition, we do an exact match 😃 

Thoughts @dg76 ?

Reply:
Oh, you are right, I didn't see that there is only one email. I used some code for searching the correct groups from the add_tester function as an example. Yes, then let's better use an exact match. So maybe like this?

```suggestion
        next unless email_param == tester.email
```

Reply:
That's fine by me, but note that that implies the email argument should take just one email, so the docs and comments will need to be updated 😊

Reply:
I have added docs and updated the comments now. Please see below, I have added the changes to the PR. 😊